### PR TITLE
Holaplex creator last

### PR DESCRIPTION
**Pull request by kristianeboe**
Ensure Holaplex is the last creator in the creator array when uploaded to Metadata.

1 commit is a hotfix

2 commit is a refactor that removes the holaplex from the form / application state

### CODE REVIEW

File: pages/nfts/new.tsx [hunk 1]
```diff
@@ -7,15 +7,17 @@ import Verify from '@/modules/nfts/components/wizard/Verify';
...
+  HOLAPLEX_CREATOR_OBJECT,
+} from '@/modules/nfts/components/wizard/RoyaltiesCreators';
 import { WalletContext } from '@/modules/wallet';
 import PriceSummary from '@/modules/nfts/components/wizard/PriceSummary';
 import MintInProgress from '@/modules/nfts/components/wizard/MintInProgress';
 import { isNil } from 'ramda';
 import OffRampScreen from '@/modules/nfts/components/wizard/OffRamp';
 import { Connection } from '@solana/web3.js';
 
-export const MAX_CREATOR_LIMIT = 5;
```
Comment (kespinola) [inline pages/nfts/new.tsx:18]: When is this checked? With holaplex in the creators array is 5 correct?

Comment (kristianeboe) [reply pages/nfts/new.tsx:18]: The MAX_CREATOR_LIMIT is used in the Royalties page for the creators array and I changed it to 4 after I removed Holaplex from that creators array

**Review by kespinola — APPROVED** [review]

**Review by kespinola — APPROVED** [review]

**Comment by kespinola** [comment]
Test works where NFT minted from holaplex.com shows up under the artworks and we are the second entry. Going to run another test for validations and 5 creators.

https://explorer.solana.com/address/5bfkQHU7qZ6Ut3k6xAsMo827ZVZCq5ELDQu4oYteAumD?cluster=devnet

image

image

**Comment by kespinola** [comment]
image

Multi creators working too.

**Pull request merged**